torch/csrc/lazy/core/ir.cpp

```diff
@@ -31,18 +41,20 @@ hash_t OpKind::hash() const {
   return StringHash(op.toQualString());
 }
 
-Node::Node(OpKind op, size_t num_outputs, hash_t node_hash, hash_t dag_hash)
+Node::Node(OpKind op, size_t num_outputs, hash_t node_hash, std::function<hash_t(bool)> dag_hash_fn)
     : op_(op),
       num_outputs_(num_outputs),
       node_hash_(node_hash),
```

Comment:
did you find a reason to keep node_hash stored in Node?  Or you just didn't remove it since it seems harmless?

Reply:
Since I don't need to store node_hash_with_sizes any more, it looks not too bad. But I can remove it if we don't plan to use it in future.